pkg/util/throttle.go

```diff
@@ -26,9 +26,12 @@ type RateLimiter interface {
 	Accept()
 	// Stop stops the rate limiter, subsequent calls to CanAccept will return false
 	Stop()
+	// Saturation returns a percentage number which describes how saturated
+	// this rate limiter is.
```

Comment:
@gmarek No, it still doesn't make sense to me. All that matters is the aggregate qps. If the actions taken are bursty then that will show up as sometimes being saturated and sometimes not. Since we poll this every second and we will look at an aggregate value, I don't think we need to add any logic to consider that case to this saturation function; it should return instantaneous saturation, as it does.

IOW, bursting really doesn't change the logic because you can't sustain it-- either you'll run out of tokens eventually and it will show up in this metric OR you're not actually exceeding the QPS limit and therefore we don't care.